README.md

```diff
@@ -1 +1,65 @@
-conda-env-update
+# Paketo Conda Env Update Cloud Native Buildpack
```

Comment:
Looks like this buildpack behavior right now has a couple of quirks - like it reuses layer only if package-list.txt is checked in with the source code, or environment file is ignored if vendor director is present. Could you please document that in the README?

Reply:
Resolved